### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by L1cardo** [twostraws/Sourceful#10]
Does anyone know how to use this or is there a documentation file ?

**Comment by markusmoenig** [comment]
Would really be nice to get a snippet of code how to integrate this into SwiftUI. @L1cardo did you figure it out ?

**Comment by CollinCunningham** [comment]
I found some example usage in the [Unwrap repo](https://github.com/twostraws/Unwrap)

**Comment by L1cardo** [comment]
Found a new project that supports SwiftUI, and that is very easy to use.
https://github.com/dwarvesf/CodeViewer

**Comment by markusmoenig** [comment]
@L1cardo Yes I actually did my own wrapper implementation of the Ace editor. It's pretty easy todo using WebKit. It's just a bit sad that a JavaScript based editor is not only more powerful than anything available natively but also much faster than anything we have in Swift.

**Comment by ilyagru** [comment]
@markusmoenig Hi, do you have your wrapper open-sourced? Or maybe you could share some details with me? Actually, I'm just looking for such a code editor to add to my iOS app and I think that this kind of wrapper would be much better than UITextView-based...

**Comment by adamdahan** [comment]
More unsupported Junk from Paul 👏

**Comment by adamdahan** [comment]
No documentation, no comments, no nothing... two claps 👏

**Comment by twostraws** [comment]
Hi, @adamdahan! You've taken the time to post a mean comment here, then remove it and post something sarcastic instead. Rather than engage in such destructive behavior, you're welcome to contribute to the project – it is open source, after all.

**Comment by adamdahan** [comment]
You are right, my apologies.
